node/src/main/scala/coop/rchain/node/api/DeployGrpcService.scala

```diff
@@ -4,10 +4,11 @@ import cats._
 import com.google.protobuf.empty.Empty
 import coop.rchain.blockstorage.BlockStore
 import coop.rchain.casper.api.BlockAPI
-import coop.rchain.casper.protocol.{DeployServiceGrpc, DeployServiceResponse, DeployData, _}
+import coop.rchain.casper.protocol.{DeployData, DeployServiceGrpc, DeployServiceResponse, _}
```

Comment:
Nit pick, but doesn't having the `_` at the end mean we don't need the other things anyway?